internal/resolver/delegatingresolver/delegatingresolver.go

```diff
@@ -188,12 +188,6 @@ func (r *delegatingResolver) updateClientConnStateLocked() error {
 		return nil
 	}
 
-	// Update curState to include target resolver's state information, such as
-	// the service config, provided by the target resolver. This ensures
-	// curState contains all necessary information when passed to UpdateState.
-	// The state update is only sent after both the target and proxy resolvers
-	// have sent their updates, and curState has been updated with the combined
-	// addresses.
 	curState := *r.targetResolverState
```

Comment:
Please move this down to where you commented about it.

Reply:
Done.